WhoIsThat/WhoIsThat/Connections/RestService.cs

```diff
@@ -33,7 +34,8 @@ public async Task<List<ImageObject>> GetImageObjects()
             
             if (!response.IsSuccessStatusCode)
             {
-                throw new ManagerException(await response.Content.ReadAsStringAsync());
+                var errorContent = await response.Content.ReadAsStringAsync();
+                throw new ManagerException((JsonConvert.DeserializeObject<BadRequestModel>(errorContent)).Message);
```

Comment:
Thats nice!! Just split it into multiple lines:)

Reply:
Will do that, next branch will be called "Cleaning" or something :D

Reply:
Refactoring is the word;)

Reply:
Amen :)